src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -136,7 +162,7 @@ const SubsidiesSummary = ({
           />
         )}
       </div>
-      {searchCoursesCta && (
+      {(searchCoursesCta && !assignmentOnlyLearner) && (
```

Comment:
[clarification] Is this only removing the "Find a course" button from the subsidies box, or is this removing the entire subsidy box altogether whent he learner only has assignment(s)? 

I believe the intent is:
* Hide the subsidies box from the sidebar altogether when learner has only assignment(s)
* Only show the subsidies box when the learner has other subsidies (e.g., subscription license, coupon codes, auto-applied budget, enterprise offer).

The above is what's reflected in Alena's Figma mocks ([source](https://www.figma.com/file/8da9TyEpEzgxQld3DZE67w/LC2---Top-down-Assignment?type=design&node-id=2540-94724&mode=dev)):

"Remove subsidy box if NO additional subsidies provided. Only display help center box."

<img width="100%" alt="image" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/2828721/033daf5b-5507-4be9-a77c-124e181cd30a">

<img width="100%" alt="image" src="https://github.com/openedx/frontend-app-learner-portal-enterprise/assets/2828721/762b289b-f188-4f91-87f9-f73c0cda967b">

Reply:
@irfanuddinahmad As mentioned by Joe in Slack, the ticket was a bit misleading from the original requirements set in the Figma designs by Alena. For now in this PR, we'll be going with this implementation just to remove the "Find a course" CTA as you have it. Long term, we may re-visit Alena's original mocks.